lib/onceover/controlrepo.rb

```diff
@@ -346,8 +346,8 @@ def fixtures
 
     def hiera_config_file
       # try to find the hiera.yaml file
-      hiera_config_file = File.expand_path('./hiera.yaml', @spec_dir) if File.exist?(File.expand_path('./hiera.yaml', @spec_dir))
       hiera_config_file = File.expand_path('./hiera.yaml', @root)     if File.exist?(File.expand_path('./hiera.yaml', @root))
+      hiera_config_file = File.expand_path('./hiera.yaml', @spec_dir) if File.exist?(File.expand_path('./hiera.yaml', @spec_dir))
```

Comment:
Oops. 

Can I suggest using a case statement instead, like I've done in the r10k_config_file method? That way it's more explicit and easier to reason about what's happening.